tests/evm.rs

```diff
@@ -248,7 +248,7 @@ fn ethereum_solidity_tests() {
         })
         .sum();
 
-    assert_eq!(errors, 1029);
+    assert_eq!(errors, 1030);
```

Comment:
That's a regression for evm

Reply:
Hmm yeah, fixed. It's weird that at the same time we support EVM target and we also don't. I keep forgetting about it.